contracts/modules/SuperMinterV1_1.sol

```diff
@@ -319,45 +330,60 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
 
         _incrementMinted(mode, d, p);
 
-        /* ----------- AFFILIATE AND PLATFORM FEES LOGIC ------------ */
+        /* ----------------- COMPUTE AND ACCRUE FEES ---------------- */
 
         TotalPriceAndFees memory f = _totalPriceAndFees(p.tier, d, p.quantity, p.signedPrice);
-
-        uint256 remaining; // The fee sent to the Sound Edition (i.e. artist fee).
-        bool affiliated;
+        MintedLogData memory l;
 
         unchecked {
             if (msg.value != f.total) revert WrongPayment(msg.value, f.total); // Require exact payment.
 
-            remaining = f.total - f.platformFee; // `platformFee <= total`;
-            platformFeesAccrued[d.platform] += f.platformFee; // Accrue the platform fee.
+            l.finalArtistFee = f.total - f.platformFee; // `platformFee <= total`;
+            l.finalPlatformFee = f.platformFee; // Initialize to the platform fee.
+            l.affiliate = p.to == p.affiliate ? address(0) : p.affiliate; // Yeah, we know it's left curved.
 
-            if (affiliated = _isAffiliatedWithProof(d, p.affiliate, p.affiliateProof)) {
-                remaining -= f.affiliateFee; // `affiliateFee <= remaining`.
-                affiliateFeesAccrued[p.affiliate] += f.affiliateFee; // Accrue the affiliate fee.
+            /* --------------------- AFFILIATE FEES --------------------- */
+
+            if (l.affiliated = _isAffiliatedWithProof(d, l.affiliate, p.affiliateProof)) {
```

Comment:
I'd add a comment with high level explanation:

Affiliates can be paid by:
- the artist, where the affiliateFee is a percent of the nft price
- the platform, where the affiliateIncentive is a portion of the platform flat fee 

In practice, we would use one of these modes, but they could be combined. The total finalAffiliateFee is a sum of these.